src/assets/css/main.css

```
@@ -189,3 +170,8 @@ canvas {
 .help.is-success {
   color: #16a34aff!important;
 }
+
+/* OwnPlate Side Bar */
+.sidebar .sidebar-content {
+  background-color: #ffffff!important;
```

```suggestion
  background-color: #ffffff !important;
```

こちらは prettier にかけると上記のようになるので `#ffffff` と `!import` の間にはスペースを入れるのが良いと思います。

（そうなっていない declaration は他にもいくつかありますが・・・😅）

---

ありがとうございます。確認します。